### ISSUE TITLE
Can't increase or decrease the volume

### ISSUE THREAD

**Issue by Felipefpl** [hrydgard/ppsspp#17574]
### Game or games this happens in

No game

### What area of the game / PPSSPP

I set up the numlock keys + and - to increase/decrease the volume in-game but it didnt work, i tried other keys aside the numlock ones but it didnt work.

### What should happen

I should be able to change the volume.

### Platform

Windows

### Mobile phone model or graphics card

Intel HD 2000

### PPSSPP version affected

The latest i got today (June 13, 2023)

### Last working version

No idea

### Graphics backend (3D API)

Direct3D 9

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Yeah this is confusing, the bindings for the volume are actually for the PSP's own +/- volume keys, in case a game listens to specifically for them (which would be unusual), it's not for increasing/decreasing the volume of the emulator.

So verdict is: Not a bug, but also very confusing.

Not really sure what to do here.

**Comment by Felipefpl** [comment]
> Yeah this is confusing, the bindings for the volume are actually for the PSP's own +/- volume keys, in case a game listens to specifically for them (which would be unusual), it's not for increasing/decreasing the volume of the emulator.
> 
> So verdict is: Not a bug, but also very confusing.

I understand, thanks for the info.
 
> Not really sure what to do here.

A suggestion: change this to actually increase/decrease the volume of the emu like duckstation does. ;)

**Comment by hrydgard** [comment]
Well, yes, but kinda want these bindings too, for completeness, so either way it gets confusing :) PSX doesn't have these buttons.

**Comment by Felipefpl** [comment]
> Well, yes, but kinda want these bindings too, for completeness, so either way it gets confusing :) PSX doesn't have these buttons.

It's a good reason, perhaps add new entries named "Increase/Decrease Volume of PPSSPP" then?

**Comment by Eloren1** [comment]
Yep much needed to control windows/android volume via gamepad shortcuts

**Comment by NABN00B** [comment]
> Yeah this is confusing, the bindings for the volume are actually for the PSP's own +/- volume keys, in case a game listens to specifically for them (which would be unusual), it's not for increasing/decreasing the volume of the emulator.

We had #17464 for that but it was closed without making a decision on what to do with them.

**Comment by hrydgard** [comment]
The main problem here is that on Android, an app doesn't really have the ability to control the Android system volume. So we can only potentially reduce the volume relative to that, which makes this idea not so great unfortunately.

**Comment by Eloren1** [comment]
There're already audio settings specifically for game volume, but I have to open the settings page, navigate and find that field every time

<img width="623" height="271" alt="Image" src="https://github.com/user-attachments/assets/ad849d6f-1d8a-4edd-bbc4-35d6493dae99" />
